### PR TITLE
removed need for ssh-keys, reordered set up process, and renamed directory

### PR DESCRIPTION
**Pull request by amalaquias**

### CODE REVIEW

File: nvidia-nano/README.md [hunk 1]
```diff
@@ -10,16 +10,18 @@ Please reach out to [Waggle Team](https://docs.waggle-edge.ai/docs/contact-us) w
...
+4. [Configure Ansible](./setup-guides/configure_ansible.md)
+5. [Running the Ansible Playbook](./setup-guides/running_ansible.md)
+6. [Configuring the Sensors](./setup-guides/configure_sensors.md)
+7. **Optional**: [Optional Tests to Run](./setup-guides/test_nano.md)
 
 Follow the Table of Contents in sequence to ensure proper configuration.
 
 Our instructions should be enough to set up the nano, but also included at the top of each guide are optional supplementary articles.
 
+For a non-technical overview of the process take a look at our [set up overview](https://youtu.be/83mLRyzxC4w).
```
Comment (jswantek) [inline nvidia-nano/README.md:25]: awesome!

File: nvidia-nano/setup-guides/configure_ansible.md [hunk 2]
```diff
@@ -39,22 +41,23 @@ This guide shows how to access `root` on the nano from another computer
...
+  4. Clone this repository using this command:
+        ```
+        git clone https://github.com/waggle-sensor/node-platforms.git
+        ```
+  5. Travel into the folder that is holding the Ansible Playbooks using this command:
+        ```
+        cd ~/node-platforms/nvidia-nano
+        ```
+        >Note: Everytime you run an ansible commmand make sure you are in this directory
+  6. Replace `{ip}` in the `ansible_inventory` file with your Jetson Nano's ip address using vim or any other text editor program
```
Comment (jswantek) [inline nvidia-nano/setup-guides/configure_ansible.md:54]: Is this still true?  I think u removed the `{ip}` part right?

Comment (amalaquias) [reply nvidia-nano/setup-guides/configure_ansible.md:54]: Yeah, I accidentally removed it from the inventory file. `{ip}` should replaced `0.0.0.0`.

Comment (jswantek) [reply nvidia-nano/setup-guides/configure_ansible.md:54]: will fix in a future PR

**Pull request merged**